app/assets/stylesheets/left_nav.scss.erb

```diff
@@ -211,7 +211,7 @@
   font-weight: 600;
 }
 .activer > a {
-  background: <%= EnrollRegistry[:qle_carousel].settings(:color).item %>;
+  background: var(--theme-primary-100, <%= EnrollRegistry[:qle_carousel].settings(:color).item %>);
```

Comment:
Is there a way to move away from using EnrollRegistry[:qle_carousel]?  Does that same color get defined in a stylesheet somewhere?  If possible, I think we should move away from referencing the CSS stored in EnrollRegistry.  Just a thought, maybe we can refactor this into `me.scss` and `dc.scss`?

Reply:
Yep, same color as a lot of the other spots (#007bc4). We can replace it everywhere we see it with the same `var(--theme-primary-100, #007bc4)` we've introduced (I still plan to rework the cr97 variable names, just planning to do that at the end of the registration workflow)

Reply:
https://github.com/ideacrew/enroll/pull/3351/commits/3dfb45ff458c9ce0627f0a2df1dfce05a983fd94